src/siteSettings/components/SiteSettingsPage/SiteSettingsPage.tsx

```diff
@@ -55,7 +54,7 @@ export interface SiteSettingsPageProps {
   onBack: () => void;
   onKeyAdd: () => void;
   onKeyRemove: (keyType: AuthorizationKeyType) => void;
-  onSubmit: (data: SiteSettingsPageFormData) => SubmitPromise;
+  onSubmit: (data: SiteSettingsPageFormData) => void;
```

Comment:
For consistency I think it should be also changed
```suggestion
  onSubmit: (data: SiteSettingsPageFormData) => SubmitPromise;
```